### DISCUSSION TITLE
[Feature] Ability to change location by city name, location name or country name

### DISCUSSION THREAD

**Discussion by GregTroar** [immich-app/immich#6421] [Feature Request]
### The feature

Hi,

First of all, Immich is awesome !
It would be great to have the possibility to have a search text box to be able to look for location by name rather than zooming in and searching on the map.

Thanks

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Duplicate of  #5691
